src/main/java/com/vacation_bot/repositories/VacationModelRepository.java

```diff
@@ -6,4 +6,8 @@
 /**
  * Talks to {@link VacationModel} collection in the database.
  */
-public interface VacationModelRepository extends MongoRepository<VacationModel, String> { }
+public interface VacationModelRepository extends MongoRepository<VacationModel, String> {
+
+    VacationModel findByUserId(String userId);
```

Comment:
Are you using this method just in tests? It's not a good practice to create methods on production side just for test usage. User instead `findAll()` in tests and remove this method.